src/assets/ElmAsset.js

```diff
@@ -0,0 +1,125 @@
+const path = require('path');
+const process = require('process');
+const Asset = require('../Asset');
+const fs = require('../utils/fs');
+const localRequire = require('../utils/localRequire');
+const {minify} = require('terser');
+
+class ElmAsset extends Asset {
+  constructor(name, options) {
+    super(name, options);
+    this.type = 'js';
+    this.hmrPageReload = true;
+  }
+
+  async collectDependencies() {
+    const {findAllDependencies} = await localRequire(
+      'find-elm-dependencies',
+      this.name
+    );
+    const dependencies = await findAllDependencies(this.name);
+    const packageFile = await findPackageFile(path.dirname(this.name));
```

Comment:
Change this to:
```Javascript
await this.getConfig(['elm.json'], {load: false});
```

This automatically recursively searches for the file and adds it to the deps, without reading it.

This way you can remove your helper function, cleans it up a little